src/sentry/tasks/integrations/create_comment.py

```diff
@@ -14,7 +14,11 @@
 # TODO(jess): Add more retry exclusions once ApiClients have better error handling
 @retry(exclude=(ExternalIssue.DoesNotExist, Integration.DoesNotExist))
```

Comment:
Now that the exceptions is swallowed can we remove `ExternalIssue.DoesNotExist` from the retry exclude?